metricbeat/scripts/generate_imports_helper.py

```diff
@@ -0,0 +1,29 @@
+from os.path import abspath, isdir, join
+from os import listdir
+
+comment = """Package include imports all Module and MetricSet packages so that they register
+their factories with the global registry. This package can be imported in the
+main package to automatically register all of the standard supported Metricbeat
+modules."""
+
+
+def get_importable_lines(go_beat_path):
+    import_line_module = "\t_ \"{beat_path}/module/{module}\""
```

Comment:
Would be nice of the tap and escaping magic would also be done by the global script, so only a list of paths / strings has to be passed.